static/app/components/externalIssues/ticketRuleModal.tsx

```diff
@@ -128,7 +131,7 @@ export default function TicketRuleModal({
       integrationId: instance.integration,
       query: initialConfigQuery,
     }),
-    {staleTime: Infinity, retry: false}
+    {staleTime: Infinity, retry: false, refetchOnMount: 'always'}
```

Comment:
Also need this because using the cached value that we're manually editing is very easy to become stale and use values that aren't applicable on the latest form.